port/unix/protect_helpers.c

```diff
@@ -45,6 +46,12 @@ struct {
 intptr_t
 protect_memory(struct OMRPortLibrary *portLibrary, void *address, uintptr_t length, uintptr_t flags)
 {
+#if defined(OMRZTPF)
+	/* Return success, zTPF does not support mprotect. */
+	IDATA rc = 0;
+
+	return rc;
```

Comment:
@pdbain-ibm Is the right answer to return success or failure when the feature is not supported?